doc/user-manual/getting-started/installation.rst

```diff
@@ -246,6 +246,12 @@ The standard library is available in Debian and Ubuntu from Lucid onwards. To in
 
   apt-get install agda-stdlib
 
+You may need to install the main agda package on Ubuntu as well
+
+.. code-block:: bash
+
+  apt-get install agda
```

Comment:
I think this paragraph should precede the instructions for the standard library.
I suggest to simply merge this into the code above, i.e.,
```
apt-get install agda agda-mode
```